### PR TITLE
add winforms and wpf dependencies causing coherency errors in installer

### PR DESCRIPTION
**Pull request by riarenas**
Trying to unblock dependency updates to installer: https://github.com/dotnet/core-eng/issues/11990

### CODE REVIEW

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -129,6 +129,14 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>82bf84c67949ab91f248e940428cfbd5c09a9e2f</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.Dotnet.WinForms.ProjectTemplates" Version="6.0.0-preview.2.21078.2">
```
Comment (riarenas) [inline eng/Version.Details.xml:132]: Should this also have the microsoft.private.winforms CPD?

Comment (mmitche) [reply eng/Version.Details.xml:132]: Should have a CPD against WPF https://github.com/dotnet/windowsdesktop/pull/1401/files#diff-fb62e94a1d6f29f863e3d0a22aa38269f6cd1d7f03b109dc06e2cbf2548b86d3L104

Comment (riarenas) [reply eng/Version.Details.xml:132]: Yep that would make sense. Thanks

**Comment by riarenas** [comment]
@mmitche anything else I'm missing here?

**Review by RussKie — APPROVED** [review]

**Pull request merged**